### ISSUE TITLE
Intermittent Bug "'list' object is not callable

### ISSUE THREAD

**Issue by denised** [ProjectDrawdown/solutions#436]
I've occasionally seen this error, but haven't been able to reliably reproduce it:  The call to `adoptiondata.make_adoption_config()` fails with the error "list object is not callable".  Previously I had seen it in Jupyter Notebook, but just now in a test run, tests for both walkable cities and waveandtidal failed with this error.  Iimmediately rerunning the tests on just those solutions succeeded without that error.
What caught my eye is that the failure occurred in `test_loader`, which doesn't usually fail.

Example of error from pytest run:
```
_____________________________________________________ test_loader _____________________________________________________
solution\waveandtidal\tests\test_waveandtidal.py:20: in test_loader
    pds1 = factory.load_scenario(solution_name,"PDS1")
solution\factory.py:37: in load_scenario
    return m.Scenario(scenario)
solution\waveandtidal\__init__.py:153: in __init__
    self.initialize_adoption_bases()
model\scenario.py:156: in initialize_adoption_bases
    adconfig = adoptiondata.make_adoption_config(overrides=overrides)
E   TypeError: 'list' object is not callable
```

**Comment by denised** [comment]
Rerunning the full test, however, did reproduce the same two failures.

**Comment by denised** [comment]
@danielmk Let me know if you see this happening.

**Comment by danielmk** [comment]
Yes I saw this bug in walkablecities. Just creating the scenario causes it:
```
from solution import factory
pds1 = factory.load_scenario("walkablecities", "PDS1")
```

The issue seems to be that `adoptiondata.make_adoption_config` is a list but I did not understand why that would ever be the case. It is clearly defined as a function in `adoptiondata` and is imported in `scenario.py`. I also did not understand why only a few scenarios are affected.

**Comment by denised** [comment]
Found it.  Cute bug.  In a completely different solution (tropicaltreestaples), a line that was supposed to look like this:
```
adconfig = adoptiondata.make_adoption_config(overrides=[ ... stuff
```
instead had a severe typo, and ended up looking like this:
```
adconfig = adoptiondata.make_adoption_config =[ ... stuff
```
The effect was to overwrite the function `adoptiondata.make_adoption_config` with a list, and solution `walkablecities` just happened to be the next time that `make_adoption_config` was called.

Fixed in e1945e1
